docs/src/tutorials/exporting.md

```diff
@@ -1,51 +1,173 @@
+
+Environment variables beginning with:
 
-## Get the environment variables
+- `env:` are exposed [as Unix environment variables](../development/variables/_index.md#top-level-environment-variables).
+- `php:` are interpreted [as `php.ini` directives](../development/variables/_index.md#php-specific-variables).
 
-If your project uses some environment variable (tokens, ...) it can be helpful to backup them if you didn't store them separately.
+All other variables are [part of `$PLATFORM_VARIABLES`](../development/variables/use-variables.md#use-platformsh-provided-variables).
 
-As stated in the Console, several possibilities exist for the environment variables.
+To back up your environment variables:
 
-* Variables beginning with `env:` are exposed as Unix environment variables
-* Variables beginning with `php:` are interpreted as `php.ini` directives.
-* All other variables are part of the environment `PLATFORM_VARIABLES` variable
+{{< codetabs >}}
+
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+
+1. Get the variable's values with:
+
+   ```bash
+   platform ssh --project {{< variable "PROJECT_ID" >}} --environment {{< variable "ENVIRONMENT" >}} -- 'echo $PLATFORM_VARIABLES | base64 -d | jq'
+   ```
+
+   Note that you can also get all the environment variable values with:
+
+   ```bash
+   platform ssh --project {{< variable "PROJECT_ID" >}} --environment {{< variable "ENVIRONMENT" >}} -- env
+   ```
+
+2. Store the data on your local computer, for example in an encrypted text file for sensitive tokens.
+
+<--->
+
+---
+title=In the Console
+file=none
+highlight=false
+---
 
-More details can be found [on the environment page](https://docs.platform.sh/administration/web/configure-environment.html#variables)
+1. In the [Console](https://console.platform.sh/), open your project and click **{{< icon settings >}}**.
+2. Click **Project Settings {{< icon chevron >}}**.
+3. Click **Variables** and access your variable's values and settings.
+4. Store the data on your local computer, for example in an encrypted text file for sensitive tokens.
 
-You can access the content of the environment variable through the Console unless the `--sensitive true` flag was set.
+Note that in the Console, you can't access the value of variables that have been created using the [`--sensitive true` flag](../development/variables/set-variables.md#variable-options).
+Use the CLI instead to retrieve these values.
 
-In that case, you can run:
-`platform ssh -p <project id> -e <environment>`
-to access all the environment variable values
+{{< /codetabs >}}
 
-and `platform ssh -p <project id> -e <environment> "echo \$PLATFORM_VARIABLES | base64 -d | jq"` to access the `PLATFORM_VARIABLES`'s values.
+## What's next
 
- 
+- Set up your [local development environment](../development/local/_index.md)
+- Migrate to [another region](../projects/region-migration.md)
```

Comment:
:clipboard: 
```suggestion
- Set up your [local development environment](../development/local/_index.md).
- Migrate to [another region](../projects/region-migration.md).
```

Reply:
Not totally sure what the first has to do with this. More context might help?

Reply:
Seems like it'd make sense to include the guide mentioned in the issue: https://docs.platform.sh/tutorials/migrating.html